.gitignore

```diff
@@ -1,6 +1,4 @@
-build/
-certs/
```

Comment:
I use this locally

Reply:
> I use this locally (certs)

Is it in the project root, or scattered around subdirs?

Reply:
can edit `.git/info/exclude` and it will act as a `.gitignore` for that clone/checkout of the repo.  I end up putting stuff like ctags files and helper scripts in there all the time.

Reply:
@jagerman I stash a folder of certs for testing in `/tests/certs/*`, and my build script copies it into the right place for running the tests. It's not a huge loss if we drop it, tom's idea of adding it to the global gitignore is good :+1:

Reply:
Any reason not to just generate those on the fly in that script, using `utils/gen-certs.sh`?